index.js

```diff
@@ -456,6 +456,42 @@ require('yargs')
                 Logger.error(err.message);
             });
     })
+    .command('extract-decorators','extracts the decorator command sets and vocabularies from a list of model files', yargs =>{
+        yargs.demandOption('models','Please provide a model');
+        yargs.option('models', {
+            describe: 'The file location of the source models',
+            alias: 'model',
+            type: 'string',
+            array:true,
+        });
+        yargs.option('locale', {
+            describe: 'The locale for extracted vocabularies',
+            type: 'string',
+            default:'en'
+        });
+        yargs.option('removeDecoratorsFromSource', {
+            describe: 'Flag to determine whether to remove decorators from source model',
+            type: 'boolean',
+            default: false
+        });
+        yargs.option('output', {
+            describe: 'output directory path',
+            type: 'string',
+        });
+    }, argv => {
+        const options = {
+            locale: argv.locale,
+            removeDecoratorsFromModel: argv.removeDecoratorsFromSource,
+            output: argv.output
+        };
+        return Commands.extractDecorators(argv.models,options)
+            .then(result => {
+                Logger.info(result);
```

Comment:
Do we always want to log the result? Even if we output to disk?

Also, what's the output format? A stringified JSON?

Reply:
It is a JSON currently. 
shall I include another option, for example `--console`, to determine whether to log or not?
We can keep one of `--output` and `--console` as mandatory.

Reply:
Given that there are multiple outputs to this command, I don't think that it makes sense to output content to the console. This is the same behaviour that we have with the compile command for example.

Reply:
Updated the PR